### DISCUSSION TITLE
typo

### DISCUSSION THREAD

**Discussion by eugene0928** [ismoilovdevml/devops-journey#15] [Q&A] [answered]
**REPO_NAME**  --> `IMAGE_NAME` bob qobti

https://github.com/ismoilovdevml/devops-journey/blob/14e0f344ce1024a1ca671678a0d7de41bd06671f/pages/guides/ci-cd/github-actions-ci-cd.en-UZ.mdx#L99

**Comment by ismoilovdevml** [comment, marked as answer]
to'grilab chiqarvordim raxmat )
